lib/large_object_store.rb

```diff
@@ -81,7 +84,13 @@ def fetch(key, options={})
     end
 
     def delete(key)
-      @store.delete("#{key}_0")
+      @store.delete(key(key, 0))
+    end
+
+    private
+
+    def key(key, i)
+      "#{key}_#{CACHE_VERSION}_#{i}"
```

Comment:
Maybe `"#{key}_v#{CACHE_VERSION}_#{i}"` so that people inspecting the keys can see it's a version?